Problem3.java

```diff
@@ -0,0 +1,18 @@
+import java.util.Stack;
+
+public class Problem3 {
+    public static String game(Stack<Character> jenny, Stack<Character> stephanie) {
+        String result = "";
+        while (!jenny.isEmpty() || !stephanie.isEmpty()) {
+            if (jenny.isEmpty())
```

Comment:
Is there a way to reduce the if-else statements? It makes the code a bit hard to read.

Reply:
To reduce them, I don't think so. Because it is needed to check if the stack is empty before doing the peek() operation, which will throw an exception if the stack is empty. Still, it is possible to place them out of loop, but additional code is needed. Also, the time complexity is a good one, O(MAX(stack1.size, stack2.size). 
Indeed, it is hard to read, but they are necessary, from my point of view.